### PR TITLE
Feat/update domain names

### PR DESCRIPTION
**Pull request by EdwardBrunton**
### Short summary
Updated domain names and added new columns to handover and swcr

Link to issue:
Closes https://github.com/equinor/cc-components/issues/882
Closes https://github.com/equinor/cc-components/issues/1115

### PR Checklist
- [x] I have performed a self-review of my own code
- [x] I have written a short summary of my changes in the PR
- [x] I have linked related issue to the PR

> [!TIP]
> To deploy the PR to Test, use the [Manual deploy fusion app TEST🚀](https://github.com/equinor/cc-components/actions/workflows/manual-deploy.yml) action.
> Remember to deploy any backend changes to Test as well!

> [!CAUTION]
> ⛔ I understand by merging my PR, the changes will be deployed to production immediately ⛔

### CODE REVIEW

File: libs/swcrapp/src/lib/config/tableConfig.tsx [hunk 1]
```diff
@@ -153,42 +153,80 @@ const columnDefinitions: ColDef<SwcrPackage>[] = [
...
+    headerTooltip: domainNames.system,
+    valueGetter: (pkg) => pkg.data?.system,
+    width: 200,
+  },
+  {
+    colId: 'priority1',
+    headerName: domainNames.commPriority1,
+    headerTooltip: domainNames.commPriority1,
+    cellRenderer: (props: ICellRendererProps<SwcrPackage, string>) => {
+      console.log(props.data?.priority1, props.data);
```
Comment (kjellhaaland) [inline libs/swcrapp/src/lib/config/tableConfig.tsx:206]: We should remove this console.log before going to production

File: libs/workorderapp/src/lib/config/tableConfig.tsx [hunk 2]
```diff
@@ -196,14 +196,34 @@ const columnDefinitions: [ColDef<WorkOrder>, ...ColDef<WorkOrder>[]] = [
...
+    headerTooltip: domainNames.commSystems,
     valueGetter: (pkg) => pkg.data?.systems,
   },
+  {
+    colId: 'EstimatedHours',
+    headerName: domainNames.remainingManHours,
+    headerTooltip: domainNames.remainingManHours,
+    valueGetter: (pkg) => pkg.data?.remainingHours,
+    cellRenderer: (props: ICellRendererProps<WorkOrder, string>) => {
+      if (!props.value) return null;
```
Comment (kjellhaaland) [inline libs/workorderapp/src/lib/config/tableConfig.tsx:215]: Why are we having a guard clause and handeling the null value later on?
Can we remove the guard clause?

File: libs/workorderapp/src/lib/config/tableConfig.tsx [hunk 3]
```diff
@@ -196,14 +196,34 @@ const columnDefinitions: [ColDef<WorkOrder>, ...ColDef<WorkOrder>[]] = [
...
+      return <StyledMonospace>{props.value ?? null}</StyledMonospace>;
+    },
+  },
+  {
+    colId: 'RemainingHours',
+    headerName: domainNames.estimatedManHours,
+    headerTooltip: domainNames.estimatedManHours,
+    valueGetter: (pkg) => pkg.data?.estimatedHours,
+    cellRenderer: (props: ICellRendererProps<WorkOrder, string>) => {
```
Comment (kjellhaaland) [inline libs/workorderapp/src/lib/config/tableConfig.tsx:224]: Why are we having a guard clause and handeling the null value later on?
Can we remove the guard clause?

**Review by kjellhaaland — CHANGES_REQUESTED** [review]
Great work! 
Left some comments to be resolved before merge :)

File: libs/workorderapp/src/lib/config/tableConfig.tsx (hunk 2 above)
Comment (EdwardBrunton) [reply libs/workorderapp/src/lib/config/tableConfig.tsx:215]: It is to prevent the table from displaying 0

**Review by kjellhaaland — APPROVED** [review]

**Pull request merged**